ecology.yaml

```diff
@@ -0,0 +1,515 @@
+
+  - name: jq
+    owner: iuc
+    tool_panel_section_label: Text Manipulation
+
+  - name: column_maker
+    owner: devteam
+    tool_panel_section_label: Text Manipulation
+
+  - name: coverage
+    owner: devteam
+    tool_panel_section_label: Text Manipulation
+
+  - name: tables_arithmetic_operations
+    owner: devteam
+    tool_panel_section_label: Text Manipulation
+
+  - name: merge
+    owner: devteam
+    tool_panel_section_label: Text Manipulation
+
+  - name: intersect
+    owner: devteam
+    tool_panel_section_label: Text Manipulation
+
+  - name: flanking_features
+    owner: devteam
+    tool_panel_section_label: Text Manipulation
+
+  - name: join
+    owner: devteam
+    tool_panel_section_label: Text Manipulation
+
+  - name: cluster
+    owner: devteam
+    tool_panel_section_label: Text Manipulation
+
+  - name: get_flanks
+    owner: devteam
+    tool_panel_section_label: Text Manipulation
+
+  - name: complement
+    owner: devteam
+    tool_panel_section_label: Text Manipulation
+
+  - name: subtract
+    owner: devteam
+    tool_panel_section_label: Text Manipulation
+
+  - name: basecoverage
+    owner: devteam
+    tool_panel_section_label: Text Manipulation
+
+  - name: subtract_query
+    owner: devteam
+    tool_panel_section_label: Text Manipulation
+
+  - name: dna_filtering
+    owner: devteam
+    tool_panel_section_label: Text Manipulation
+
+
+  - name: column_order_header_sort
+    owner: iuc
+    tool_panel_section_label: Text Manipulation
+
+  - name: column_remove_by_header
+    owner: iuc
+    tool_panel_section_label: Text Manipulation
+
+  - name: metagenomeseq_normalization
+    owner: iuc
+    tool_panel_section_label: Text Manipulation
+
+  - name: gff3_rebase
+    owner: iuc
+    tool_panel_section_label: Text Manipulation
+
+
+
+  - name: column_arrange_by_header
+    owner: bgruening
+    tool_panel_section_label: Filter and Sort
+  - name: seq_filter_by_id
+    owner: peterjc
+    tool_panel_section_label: Filter and Sort
+  - name: find_in_reference
+    owner: jjohnson
+    tool_panel_section_label: Filter and Sort
+  - name: sample_seqs
+    owner: peterjc
+    tool_panel_section_label: Filter and Sort
+  - name: collection_column_join
+    owner: iuc
+    tool_panel_section_label: Join, Subtract and Group
+
+  # - install_repository_dependencies: true
+  #   install_resolver_dependencies: true
+  #   install_tool_dependencies: false
+  #   name: datamash_wrapper
+  #   owner: agordon
+  #   tool_panel_section_label: Join, Subtract and Group
+
+  - name: suite_datamash
+    owner: iuc
+    tool_panel_section_label: Join, Subtract and Group
+
+
+  - name: subtract
+    owner: devteam
+    tool_panel_section_label: Join, Subtract and Group
+
+
+  - name: collection_column_join
+    owner: iuc
+    tool_panel_section_label: Join, Subtract and Group
+
+  - name: suite_datamash
+    owner: iuc
+    tool_panel_section_label: Join, Subtract and Group
+
+
+  - name: subtract
```

Comment:
This is also multiple times included.

Reply:
Double eagle eye! Thank you! And sorry again for that....